.nvmrc

```diff
@@ -1 +1 @@
-v8.9.4
+v8.10.0
```

Comment:
Do we need any additional setup for travis? Or will it base it of the `.nvmrc` ?

Reply:
Travis uses the `nvmrc` :+1:

Reply:
👌